tests/integration/install_test.go

```diff
@@ -111,3 +111,26 @@ func TestInstall_withDockerignore(t *testing.T) {
 	// We should check this once https://github.com/cnabio/cnab-go/issues/78 is closed
 	require.EqualError(t, err, "1 error occurred:\n\t* container exit code: 1, message: <nil>\n\n")
 }
+
+func TestInstall_stringParam(t *testing.T) {
+	t.Parallel()
+
+	p := porter.NewTestPorter(t)
+	defer p.Teardown()
+	p.SetupIntegrationTest()
+	p.Debug = false
+
+	p.AddTestBundleDir("testdata/bundles/bundle-with-string-params", false)
+
+	installOpts := porter.NewInstallOptions()
+	installOpts.Params = []string{"name=Demo Time"}
```

Comment:
```suggestion
	installOpts.Params = []string{"name=\"Demo Time\""}
```

Reply:
@Simpcyclassy try this, the problem I believe is that our test doesn't have quotes around the parameter. Rereading the issue this is expected https://github.com/getporter/porter/issues/1862

As this should be the given command: `$porter install -r getporter/hello-llama:v0.1.1 --param name="demo time"`

At least for me locally this passes the tests :tada: and from reading this again as I understand this is the intended behavior. Using single and double quotes catches both this behavior but also when no quotes were passed so that is incorrect as that would catch potentially other flags passed to the command.

Reply:
Hmm on further inspection perhaps this is not a good solution. I am running the code locally and it is not passing through "demo time" only "demo" :(

Reply:
:warning:  ignore this